pontoon/base/models.py

```diff
@@ -890,18 +891,17 @@ class Repository(models.Model):
     project = models.ForeignKey(Project, related_name='repositories')
     type = models.CharField(
         max_length=255,
-        blank=False,
         default='git',
         choices=TYPE_CHOICES
     )
-    url = models.CharField("URL", max_length=2000, blank=True)
-
-    """
-    Prefix of the resource URL, used for direct downloads. To form a full
-    URL, relative path must be appended.
-    """
-    permalink_prefix = models.CharField("Permalink prefix", max_length=2000, blank=True)
+    url = models.CharField("URL", max_length=2000)
```

Comment:
Sadly we can't. I remember trying that, but it didn't recognize the GitHub SSH-based URLs as valid.